src/client/automation/playback/scroll.js

```diff
@@ -47,33 +58,44 @@ export default class ScrollAutomation {
         styleUtils.setScrollTop(element, top);
     }
 
-    static _getScrollToPoint (elementDimensions, { x, y }) {
+    _forceMaxScrollMargin (newMaxScrollMargin) {
```

Comment:
I'm not sure that `force` is suitable word here, how you can force margin?
Force scrolling or increase margin, something like this I guess.